README.md

```diff
@@ -4,7 +4,7 @@
 * Add `management.security.enabled: false` property into `application.yml` file if you need to connect to secured endpoints from actuator (don't commit that configuration) 
 
 # Prepare plugins
-Download the petclinic-plugins project and run: gradle publishToMavenLocal (publishing plugins to local maven repository)
+Download the petclinic-plugins and read readme.md
```

Comment:
Good... Where can the person who is not familiar with our projects find the project with plugins?